.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+# Pull Request
+
+## Summary
+
+Provide a concise and clear summary of your pull request.
+
+## Motivation
```

Comment:
suggestion: I believe the "Motion" isn't needed in favor of the Related Issues/PRs which by itself contains the reason behind the PR. What do you think?

Reply:
Feels the same now. Makes it much more minimal.